### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by Madhan-SWE**
Readme updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,110 @@
...
+  * Enable `kube-apiserver` feature gates `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true`
+  * Enable `kubelet` feature gates `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true`
+
+
+## Installation
+#### Set up driver permission
+
+* Using secret object - Generate IBMCLOUD_APIKEY from the UI, put that user's credentials in [secret manifest](../deploy/kubernetes/secret.yaml), then deploy the secret
+```sh
+curl https://raw.githubusercontent.com/kubernetes-sigs/ibm-powervs-block-csi-driver/blob/main/deploy/kubernetes/secret.yaml > secret.yaml edit the IBMCLOUD_API_KEY
```
Comment (mkumatag) [inline README.md:65]: `edit the IBMCLOUD_API_KEY` can be shifted to next line with `#` prefixed

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Madhan-SWE) [reply README.md]: resolved

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Comment by mkumatag** [comment]
Merging this manually for now.

**Comment by Rajalakshmi-Girish** [comment]
image
It says `In merge pool` but is not being picked up.

**Comment by Rajalakshmi-Girish** [comment]
@mkumatag how did it merge? Did you do anything

**Comment by mkumatag** [comment]
> @mkumatag how did it merge? Did you do anything

I merged it manually,!

**Comment by Rajalakshmi-Girish** [comment]
> > @mkumatag how did it merge? Did you do anything
> 
> I merged it manually,!

Looks like you also tried re-creating tide.

**Pull request merged**